go.mod

```diff
@@ -3,8 +3,9 @@ module github.com/ethersphere/beekeeper
 go 1.16
 
 require (
-	github.com/ethersphere/bee v0.5.3
+	github.com/ethersphere/bee v1.0.0
```

Comment:
it would be good to make sure that the integration tests won't fail once we merge this branch and tag it. maybe to run the ci tests in the bee repo against this branch

Reply:
ok, let me run them in my cluster.

Reply:
Tests pass 👍 , Merging